DataFormats/CTPPSReco/interface/CTPPSPixelLocalTrack.h

```diff
@@ -73,6 +73,14 @@ class CTPPSPixelLocalTrack
   
   enum TrackPar {x0 = 0, y0 = 1, tx = 2, ty = 3}; 
 
+  /// Track information byte for bx-shifted runs: 
+  /// reco_info = notShiftedRun    -> Default value for tracks reconstructed in non-bx-shifted ROCs
+  /// reco_info = allShiftedPlanes -> Track reconstructed in a bx-shifted ROC with bx-shifted planes only
+  /// reco_info = noShiftedPlanes  -> Track reconstructed in a bx-shifted ROC with non-bx-shifted planes only
+  /// reco_info = mixedPlanes      -> Track reconstructed in a bx-shifted ROC both with bx-shifted and non-bx-shifted planes
+  /// reco_info = invalid          -> Dummy value. Assigned when reco_info is not computed
+  enum reconstructionInfo: unsigned short {notShiftedRun = 0, allShiftedPlanes = 1, noShiftedPlanes = 2, mixedPlanes = 3, invalid = 5};
```

Comment:
This info is meaningful only for pixel tracks, where this bx-shift took place. There is no need to define it for diamond tracks, since all diamond planes are time-aligned. 

I'm changing the name and switching to scoped enum